src/verification.cpp

```diff
@@ -15,3 +15,21 @@ void cloyster::checkEffectiveUserId()
             "This program must be run with root privileges");
     }
 }
+
+#ifdef BUILD_TESTING
+#include <cloysterhpc/tests.h>
+
+TEST_SUITE("Test user privileges")
```

Comment:
I don't get this one. This should be a runtime check and not effectively a test. Because it doesn't matter if it's compiled as root or not, but execution should be done with root.